makefile

```diff
@@ -9,16 +9,28 @@ LIBS = -lwsock32
 else
 OPTS = -std=gnu99 -Wall -W -O3 
 endif
+DEBUG_OPTS = -DNTRIPCLIENT_DEBUG_LOG
```

Comment:
Does this change the process of compilation of ntripclient with debug enabled?

Reply:
No.

The regular compilation is built as:

`make`

Previous to this change, we built the debug version by #define NTRIPCLIENT_DEBUG_LOG in ntripclient.c.  With this change, we simply make the debug version with:

`make debug`

Reply:
ah, got it. could you please add this to the readme?

Reply:
Done!